src/R/plot_detection_curve.R

```diff
@@ -266,24 +278,34 @@ plot_detection_curve <- function(dist_model, design_mat = NULL, labels = NULL,
                  width = trunc / n_breaks, fill = "white", color = "black") +
         geom_line(data = df_gemiddelde, aes(x = dist, y = lineval), size = 1,
                   linetype = "dashed") +
-        scale_y_continuous(breaks = seq(0, 1, 0.2)) +
+        scale_y_continuous(limits = c(0, NA),
+                           breaks = seq(0, 100, by = 0.25)) +
         labs(x = "Afstand (m)", y = "Detectiekans") +
         theme(legend.position = c(1, 1),
               legend.justification = c(1, 1),
-              legend.background = element_rect(fill = "white", color = "black"))
+              legend.background = element_rect(fill = "white", color = "white"))
     # Plot bar graph
     } else {
       out <- ggplot(hist_df) +
         geom_bar(aes(x = mids, y = counts), stat = "identity",
                  width = trunc / n_breaks, fill = "white", color = "black") +
-        scale_y_continuous(breaks = seq(0, 1, 0.2)) +
+        scale_y_continuous(limits = c(0, NA),
+                           breaks = seq(0, 100, by = 0.25)) +
         labs(x = "Afstand (m)", y = "Detectiekans") +
         theme(legend.position = c(1, 1),
               legend.justification = c(1, 1),
-              legend.background = element_rect(fill = "white", color = "black"))
+              legend.background = element_rect(fill = "white", color = "white"))
     }
   }
 
+  if (show_data) {
+    distance_data <- data.frame(dist = sort(unique(dist_model$ddf$data$distance)))
+    out <- out +
+      geom_point(data = distance_data, aes(x = dist,
+                                           y = rep(0, nrow(distance_data))),
+                 shape = "|", col = "brown", size = 3)
```

Comment:
This works, but ggplot has a geom for this. See https://ggplot2.tidyverse.org/reference/geom_rug.html

Reply:
For some reason I could not find that geom! I just tested it and I think the position looks better with geom_point (lines too low with geom_rug).